fiorella/src/snippets/navbar-elements.liquid

```diff
@@ -0,0 +1,5 @@
+<ul>
+  {% for collection in collections %}
+    <li><a href="">{{ collection.title }}</a></li>
```

Comment:
>you forgetting set attribute `href="{{ collection.url }}"`

Reply:
Done! I already push another commit for a new ticket "Navigation bar" to this pull request, since it belongs to the header I thought  it would be better to be in this branch, so I appreciate a Review on that commits. Thanks!